packages/offix-datasync-codegen/tests/__snapshots__/DataSyncClientPlugin.test.ts.snap

```diff
@@ -0,0 +1,71 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Test datasync config 1`] = `
```

Comment:
Not sure if we need to generate config or it should be just part of the docs.

Reply:
We can generate but still leave it editable to the user. Like generate boilerplate code. To ensure that we don't overwrite the user's work, we can add a config option to the plugin to skip generating datasync config

Reply:
Trick is to not generate anything that should be editable, but ok to leave it